packages/contracts-bedrock/contracts/L2/L2StandardBridge.sol

```diff
@@ -187,7 +185,79 @@ contract L2StandardBridge is StandardBridge, Semver {
         } else {
             _initiateBridgeERC20(_l2Token, l1Token, _from, _to, _amount, _minGasLimit, _extraData);
         }
+    }
+
```

Comment:
my one nit here is that this comment doesn't follow the same standards as other comments that follow natspec

Reply:
Would it address you feedback if

Reply:
whoops, didn't mean to hit send on that. :P

I didn't realize that you could add docs on top of using `@inheritdoc`, good call it's much nicer now.